crates/assists/src/tests.rs

```diff
@@ -80,10 +80,8 @@ fn check_doc_test(assist_id: &str, before: &str, after: &str) {
     let actual = {
         let source_change = assist.source_change.unwrap();
         let mut actual = before;
-        for source_file_edit in source_change.source_file_edits {
-            if source_file_edit.file_id == file_id {
-                source_file_edit.edit.apply(&mut actual)
-            }
+        if let Some(source_file_edit) = source_change.source_file_edits.edits.get(&file_id) {
```

Comment:
```suggestion
        if let Some(source_file_edit) = source_change.source_file_edits.get(file_id) {
```

I generally find time spend on nice interfaces to be spend well